src/control/controlobjectscript.h

```diff
@@ -45,4 +48,5 @@ class ControlObjectScript : public ControlProxy {
     const RuntimeLoggingCategory m_logger;
     CompressingProxy m_proxy;
     bool m_skipSuperseded; // This flag is combined for all connections of this Control Object
+    bool m_callbackExecuting = false;
```

Comment:
Please initialize the variable in the constructor.
Initialization in the header will create a hidden static inline constructor which we want to avoid.